CMakeLists.txt

```diff
@@ -12,12 +12,13 @@ project(DuckDB)
 
 find_package(Threads REQUIRED)
 
-if (CMAKE_VERSION VERSION_LESS "3.1")
+if(NOT "${CMAKE_CXX_STANDARD}")
```

Comment:
Perhaps we should throw an error if `CMAKE_CXX_STANDARD` is set but the cmake version is less than 3.1?

Reply:
ahh right - I added this for compatibility <3.1 instead of erroring:

```
if ("${CMAKE_CXX_STANDARD}")
  if (CMAKE_VERSION VERSION_LESS "3.1")
    set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++${CMAKE_CXX_STANDARD}")
  endif ()
endif ()
```

but please do let me know if you prefer an error.